include/sapi/tss2_mu.h

```diff
@@ -1848,6 +1848,65 @@ Tss2_MU_TPMT_TK_HASHCHECK_Unmarshal(
     size_t        *offset,
     TPMT_TK_HASHCHECK *dest);
 
+TSS2_RC Tss2_MU_TPM2_HANDLE_Marshal (const TPM2_HANDLE in,
```

Comment:
Can we keep the format consistent with the above please.

Reply:
fixed